plugins/janus_streaming.c

```diff
@@ -7166,6 +7197,8 @@ janus_streaming_mountpoint *janus_streaming_create_rtsp_source(
 	live_rtsp_source->rtsp_url = g_strdup(url);
 	live_rtsp_source->rtsp_username = username ? g_strdup(username) : NULL;
 	live_rtsp_source->rtsp_password = password ? g_strdup(password) : NULL;
+	live_rtsp_source->rtsp_stream_uri = NULL;
+	live_rtsp_source->rtsp_quirk = FALSE;
```

Comment:
Shouldn't this be equal to the `quirk` that's been passed to `janus_streaming_create_rtsp_source`?